src/ai/ai.cpp

```diff
@@ -1677,11 +1696,10 @@ dcon::cb_type_id pick_fabrication_type(sys::state& state, dcon::nation_id from,
 			continue;
 		if((bits & (military::cb_flag::po_demand_state | military::cb_flag::po_annex)) == 0)
 			continue;
-		if(military::cb_infamy(state, c) > 13.0f)
+		if(state.world.nation_get_infamy(from) + military::cb_infamy(state, c) > state.defines.badboy_limit)
```

Comment:
13 was picked intentionally to prevent the ai from spending all its infamy in one place

Reply:
25/2 = 12.5

Reply:
Yes. But with your change, if the nation has 0 infamy, it could spend up to 25 fabricating a single cb. I would rather have the nation not spend all its infamy in one place